svelte/rollup.config.ts

```diff
@@ -37,6 +37,17 @@ const envKeys = () => {
   );
 };
 
+const addSyntheticNamedExportsToSkippedNodeImports = (): Plugin => ({
+  load: (importee) => {
```

Comment:
add comment and ref link on why this addition

Reply:
done